src/clients/c++/perf_client/perf_client.cc

```diff
@@ -118,144 +119,32 @@ SignalHandler(int signum)
 //     of "throughput, latency, concurrent request count" tuples will be
 //     reported in increasing load level order.
 //
+// - Maintaining Target Request Rate:
+//     Unlike above, here the client will maintain a fixed rate of requests
```

Comment:
You should say something here about the distribution. The rate of requests is fixed over some time window but within that window the rate of requests varies based on a selected distribution.